src/main/java/ArgumentException.java

```diff
@@ -0,0 +1,43 @@
+public class ArgumentException extends DukeException {
+
+    static final String MESSAGE = "Wrong Arguments Specified: ";
+
+    /**
+     * Error type according to the list below:
+     * <p> 1. Format error of a todo-task </p>
+     * <p> 2. Format error of a deadline-task </p>
+     * <p> 3. Format error of a event-task </p>
+     * <p> 4. Format error of a done or delete command </p>
+     */
+    protected int type;
+
+    /**
+     * Initializes a newly created ArgumentException object with the error type.
+     * @param type Error type according to the list below:
+     *             <p> 1. Format error of a todo-task </p>
+     *             <p> 2. Format error of a deadline-task </p>
+     *             <p> 3. Format error of a event-task </p>
+     *             <p> 4. Format error of a done or delete command </p>
+     */
+    public ArgumentException(int type) {
+        super(MESSAGE);
+        this.type = type;
+    }
+
+    /**
+     * Converts this object to a string that represents the error message
+     * @return A string representing the error message depending on the specific error it is associated with
+     */
+    @Override
+    public String toString() {
+        if (this.type == 1) {
+            return super.toString() + "A todo-task should be specified as follows \n todo <task_description> \n -----------------------------------------------------";
```

Comment:
[Code Standard] Line length should be no longer than 120 chars

Reply:
Oops I missed this out when checking for my coding standards, thanks for pointing this out!